constellations/gemini-user/src/achievements/achievements.sass

```diff
@@ -1,2 +1,57 @@
 #view-achieve h1
-  color: red
+	color: red
+
+
+.achieve-card
```

Comment:
This file's tab spaces are a bit weird.
Can you make them 4-space tabs please? We'll try to make all files size 4 tabs as a standard.